src/themes/appletv/theme.css

```
@@ -23,21 +23,25 @@ html {
 .skinHeader-withBackground {
     color: rgba(0, 0, 0, 0.7);
     background: #303030;
-    background: -webkit-gradient(linear, left top, right top, from(#bcbcbc), color-stop(#a7b4b7), color-stop(#beb5a5), color-stop(#adbec2), to(#b9c7cb));
-    background: -webkit-linear-gradient(left, #bcbcbc, #a7b4b7, #beb5a5, #adbec2, #b9c7cb);
-    background: -o-linear-gradient(left, #bcbcbc, #a7b4b7, #beb5a5, #adbec2, #b9c7cb);
     background: linear-gradient(to right, #bcbcbc, #a7b4b7, #beb5a5, #adbec2, #b9c7cb);
 }
 
 .skinHeader.semiTransparent {
-    -webkit-backdrop-filter: none !important;
     backdrop-filter: none !important;
 }
 
+.layout-tv .skinHeader.semiTransparent {
+    background: none;
+}
+
 .pageTitleWithDefaultLogo {
     background-image: url(../../assets/img/banner-dark.png);
 }
 
+.layout-tv .pageTitleWithDefaultLogo {
+    background-image: url(../../assets/img/icon-transparent.png);
```

```suggestion
    background-image: url(../../assets/img/banner-light.png);
```

This is up for debate. I think it looks better with the Jellyfin text next to the logo.

---

This is meant to save horizontal space in the TV layout (for the tabs, mostly), as it's very limited due to everything being larger.

Ideally, we'll move the tab bar around to use the freed space a bit, but that requires other stuff to be merged in (the breakpoint PR, mostly)

---

Ok, that sounds good.